File: test/metabolism/demo/tmp/7_cmeans_fuzzy_clustering_analysis/scripts/quick_check.py
import csv
with open("G:/OmicsWorks/test/metabolism/demo/tmp/preprocessed_expression.csv", "r") as f:
 reader = csv.reader(f)
 for i, row in enumerate(reader):
 if i ==0:
 print("HEADER:", row[:5])
 elif i <=5:
 print(f"Row {i}: {row[:3]}")
 else:
 break
print("---")
with open("G:/OmicsWorks/test/metabolism/metabolites.csv", "r", encoding="utf-8") as f:
 reader = csv.reader(f)
 for i, row in enumerate(reader):
 if i ==0:
 print("ANNO HEADER:", row[:5])
 elif i <=5:
 print(f"Anno Row {i}: {row[:5]}")
 else:
 break

File: test/metabolism/demo/tmp/7_cmeans_fuzzy_clustering_analysis/scripts/quick_check.R
# Quick check script
expr_path <- "G:/OmicsWorks/test/metabolism/demo/tmp/preprocessed_expression.csv"
anno_path <- "G:/OmicsWorks/test/metabolism/metabolites.csv"

# Read expression
raw <- read.csv(expr_path, check.names = FALSE)
cat("Expression header:", colnames(raw)[1], "\n")
cat("First few rownames:", paste(head(raw[,1]), collapse=", "), "\n")
cat("Expression dims:", nrow(raw), "x", ncol(raw), "\n")

# Read annotation
anno <- read.csv(anno_path, check.names = FALSE)
cat("\nAnnotation column names:", colnames(anno), "\n")
cat("First few col1:", paste(head(anno[,1]), collapse=", "), "\n")
cat("First few id col:", paste(head(anno$id), collapse=", "), "\n")
